openrpc.json

```diff
@@ -766,10 +767,20 @@
             "type": "object",
             "properties": {
               "snapId": {
-                "description": "The ID of the Snap to invoke."
+                "description": "The ID of the Snap to invoke.",
+                "type": "string"
               },
               "request": {
-                "description": "The JSON-RPC request object to send to the invoked Snap."
+                "description": "The JSON-RPC request object to send to the invoked Snap.",
+                "type": "object",
+                "properties": {
+                  "method": {
+                    "type": "string"
+                  },
+                  "params": {
+                    "type": "string"
```

Comment:
```suggestion
                    "type": "object"
```

Reply:
should really be `oneOf` object or array

Reply:
Since `oneOf` is not supported I think (https://github.com/open-rpc/spec/issues/278), we'll just go with object for now?